app/models/theme.rb

```diff
@@ -601,11 +601,12 @@ def disabled_at
     find_disable_action_log&.created_at
   end
 
-  def scss_load_paths
-    return if self.extra_scss_fields.empty?
+  def with_scss_load_paths
+    return yield([]) if self.extra_scss_fields.empty?
 
-    @exporter ||= ThemeStore::ZipExporter.new(self)
-    ["#{@exporter.export_dir}/scss", "#{@exporter.export_dir}/stylesheets"]
+    ThemeStore::ZipExporter.new(self).with_export_dir(extra_scss_only: true) do |dir|
+      yield ["#{dir}/stylesheets"]
```

Comment:
Removed `/scss` because the exporter will never put files in this location. They will always be placed under `/stylesheets` thanks to the `canonical` definition here:

https://github.com/discourse/discourse/blob/657dff354494ed5906beb6fb9e2134c3e0f78792/app/models/theme_field.rb#L480-L482